### PR TITLE
Add a new section theory with various resources and add to courses

### PR DESCRIPTION
**Pull request by aarohikulkarni10**
Added a new section "E-notes" that includes various notes and books pdf

### CODE REVIEW

**Comment by kasbah** [comment]
Hey, thanks very much for collecting these. Could you please change:

- Change the style to conform to the rest of the links in the page. I.e. a link with the name and a short description of each link.
- Fix the second link, it doesn't go anywhere.
- When I read "E-Notes" it doesn't tell me anything, is there a better name for these materials maybe?

**Comment by aarohikulkarni10** [comment]
Thank you. I will do that!

**Comment by aarohikulkarni10** [comment]
Please check and let me know if it looks fine now. 😃 
Thank you.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ This list is for websites, services, software, tools and more: everything that y
 
 - [Learning](#learning)
   - [Technical Tutorials](#technical-tutorials)
-  - [Theory and Courses](#theory-and-courses)
+  - [Theory](#theory)
+  - [Courses](#courses)
```
Comment (joaoantoniocardoso) [inline README.md:18]: Can you swap the order of these two to be cohesive with the text?
```suggestion
  - [Courses](#courses)
  - [Theory](#theory)
```

**Comment by joaoantoniocardoso** [comment]
Hi @aarohikulkarni10, I think it is fine, just needs a rebase, can you do it? Thanks!

**Pull request merged**